src/ds/d400/d400-factory.cpp

```diff
@@ -649,6 +649,27 @@ namespace librealsense
 
             return tags;
         };
+
+        void hardware_reset() override 
+        {
+            d400_device::hardware_reset();
+            auto dev_info = this->get_device_info();
+            auto non_const_device_info = std::const_pointer_cast< librealsense::device_info >( dev_info );
+            std::vector< std::shared_ptr< device_info > > devices{ non_const_device_info };
+            auto ctx = get_context();
+            //creating fake notification to trigger invoke_devices_changed_callbacks, causing connection and disconnection of mipi device
```

Comment:
I would move this comment to above this section right after the base `hardware_reset` call as all other are related to the fake notification, right?

Reply:
Done